clocksilencer.ino

```diff
@@ -12,10 +12,11 @@ const char *password = "PASS";
 RTC_DATA_ATTR bool rtcTimeWasAlreadySetFromNTP = false;
 bool noWifi;
 int myDow, mySec, myMin, myHour;
+int sleepHelper;
 void setup() {
   Serial.begin(115200);
   hwSerial.begin(9600, SERIAL_8N1, 16,
-                 17); // second serial port to serve mp3 player
```

Comment:
Could you create separate pull request only for formatting?

It would be easier to understand two smaller:

1. formatting changes (no change in logic)
2. only changes for logic (behaviour)

Reply:
Ok